config/docking_server_nonav.yaml

```
@@ -0,0 +1,40 @@
+/**:
+  ros__parameters:
+    controller_frequency: 50.0
+    initial_perception_timeout: 5.0
+    wait_charge_timeout: 10.0
+    dock_approach_timeout: 20.0
+    undock_linear_tolerance: 0.19
+    undock_angular_tolerance: 0.08
+    max_retries: 3
+    base_frame: "<robot_namespace>/wibotic_receiver_link" # Distance to the dock is calculated from this frame
+    fixed_frame: "<robot_namespace>/odom"
+    dock_backwards: false
+    dock_prestaging_tolerance: 0.15
+
+
+    dock_plugins: ["charging_dock"]
+    charging_dock:
+      plugin: husarion_ugv_docking::ChargingDock
+      external_detection_timeout: 0.2
+      docking_distance_threshold: 0.04
+      docking_yaw_threshold: 0.2
+      staging_x_offset: -0.8
+      filter_coef: 0.1
+      use_wibotic_info: <use_wibotic_info_param>
+      wibotic_info_timeout: 1.0
+
+    docks: ["main"]
+    main:
+      type: charging_dock
+      frame: <robot_namespace>/odom
+      dock_frame: main_wibotic_transmitter_link
+      pose: [2.00, 1.20, 1.57] # [x, y, yaw] of the dock on the map. Used also for spawning dock in the simulation.
+      apriltag_id: 0
+
+    controller:
+      k_phi: 1.0
+      k_delta: 2.0
+      v_linear_min: 0.0
+      v_linear_max: 0.3
+      v_angular_max: 0.15
```

```suggestion
      v_linear_max: 0.1
      v_angular_max: 0.1
```
imo this can be decreased to make docking less aggressive